.github/workflows/cancel.yml

```
@@ -0,0 +1,16 @@
+name: Cancel workflows
+
+on: [push, pull_request_target]
+
+env:
+  TZ: Europe/Berlin
+
+jobs:
+  cancel:
+    name: "Cancel previous runs"
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          workflow_id: 2200049
```

```suggestion
          workflow_id: 2200049
```

I guess this is a fixed number; maybe a comment how to / where to find that number would be helpful

---

Since it's a parameter to the action, I would always suggest to just look at the action's documentation and not put everything here: https://github.com/styfle/cancel-workflow-action

---

So it is retrieved from https://api.github.com/repos/seqan/seqan3/actions/workflows

which is 

```json
{
  "total_count": 2,
  "workflows": [
    {
      "id": 2200049,
      "node_id": "MDg6V29ya2Zsb3cyMjAwMDQ5",
      "name": "SeqAn3 CI",
      "path": ".github/workflows/ci.yml",
      "state": "active",
      "created_at": "2020-08-14T13:34:09.000Z",
      "updated_at": "2020-08-14T13:34:09.000Z",
      "url": "https://api.github.com/repos/seqan/seqan3/actions/workflows/2200049",
      "html_url": "https://github.com/seqan/seqan3/blob/master/.github/workflows/ci.yml",
      "badge_url": "https://github.com/seqan/seqan3/workflows/SeqAn3%20CI/badge.svg"
    },
    {
      "id": 582073,
      "node_id": "MDg6V29ya2Zsb3c1ODIwNzM=",
      "name": "Deploy documentation (https://docs.seqan.de/)",
      "path": ".github/workflows/documentation.yaml",
      "state": "active",
      "created_at": "2020-02-17T13:42:41.000Z",
      "updated_at": "2020-05-27T19:49:41.000Z",
      "url": "https://api.github.com/repos/seqan/seqan3/actions/workflows/582073",
      "html_url": "https://github.com/seqan/seqan3/blob/master/.github/workflows/documentation.yaml",
      "badge_url": "https://github.com/seqan/seqan3/workflows/Deploy%20documentation%20(https://docs.seqan.de/)/badge.svg"
    }
  ]
}
```

which is `.github/workflows/ci.yml`

I would suggest

```suggestion
          workflow_id: 2200049 # github id for .github/workflows/ci.yml
```

---

Yes, this is OK.
I just didn't want to put documentation from the action there, because this might be subject to chance, i.e. maybe the API changes at some point, but the ID stays the same, etc.